mediapipe/pose_tracking_dll/pose_tracking.h

```diff
@@ -108,6 +108,7 @@ class DLLEXPORT PoseTracking {
   bool processFrame(const cv_wrapper::Mat& inputRGB8Bit);
   cv_wrapper::Mat lastSegmentedFrame();
   cv_wrapper::Point3f* lastDetectedLandmarks();
+  float* lastLandmarksVisibility();
```

Comment:
Can we package the landmark positions (`nimagna::cv_wrapper::Point3f`) and visibility (`float`) into a single struct? It would keep the data synchronized. I'm thinking that having separate functions to get the data might cause synchronization issues if something gets executed in-between.